CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v0.1.0] - 2022-03-30
 ### Added
 - Initial project release.
+- Option `--prompt-hint` to support custom text to prompt caller in web and WAM mode.
```

Comment:
This line should go under the unreleased section. v0.1.0 is already released ;)

Reply:
Wow. I totally missed that. Good catch, @kyle-rader.